### PR TITLE
Added time and agenda component

### PR DESCRIPTION
**Pull request by anupkgurung**
Added agenda component
Added time component

live link : https://taskyextesion.vercel.app/

### CODE REVIEW

File: src/component/Agenda/Agenda.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                    task === null ?
+                    <>
+                        <p className={`${agenda.font__size}`}>Any task in mind ? Lets get it done..</p>
+                        <Input type="text" onKeyPressHandler={setUserTask} value={task} />
+                    </>                    
+                    :
+                    <p className={`${agenda.font__size} ${agenda.flex}`}>
+                        <span className={`material-icons-outlined ${agenda.pr__1}`}>task</span>
+                        {`Task for today is ${task}`}
+                        {/* <span className={`material-icons-outlined cursor ${agenda.pointer}`}>more_horiz</span> */}
```
Comment (arpit01923) [inline src/component/Agenda/Agenda.jsx:31]: remove unnecessary data

**Review by arpit01923 — APPROVED** [review]
- clean code
- functionality works fine
- good to merge

File: src/component/Time/time.module.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    margin-bottom: -5rem
+}
+.greet__font {
+    font-size: 5rem;
+}
+.pt__1_5 {
+    padding-top: 1.5rem;
+}
+.txt__decor{
+    color: white ;
```
Comment (OmKakatkar) [inline src/component/Time/time.module.css:18]: try not to hard code color values

**Review by OmKakatkar — APPROVED** [review]
* Code is clean

**Review by manojncsoni007 — COMMENTED** [review]
Looks everything good

**Pull request merged**